src/DependencyInjection/NeustaConverterExtension.php

```diff
@@ -42,6 +42,11 @@ public function loadInternal(array $mergedConfig, ContainerBuilder $container):
     private function registerConverterConfiguration(string $id, array $config, ContainerBuilder $container): void
     {
         foreach ($config['properties'] ?? [] as $targetProperty => $sourceConfig) {
+            $nullSafety = false;
+            if (str_ends_with($targetProperty, '?')) {
+                $nullSafety = true;
+                $targetProperty = substr($targetProperty, 0, \strlen($targetProperty) - 1);
```

Comment:
The `strlen()` is not necessary if you just want to remove something from the end:

```suggestion
                $targetProperty = substr($targetProperty, 0, -1);
```